### PR TITLE
add sundararajan's results

### PR DESCRIPTION
**Pull request by sundararajan20**
* Final validation loss: 3.89
* Link: https://api.wandb.ai/links/sundararajan-team/kvqmagh7
* Description: vanilla model with 6 layers + weight tying of input and output embeddings + hyperparameter optimization

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
